app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RelationshipsController < ApplicationController
+  before_filter :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.destroy if relationship.follower == current_user
+    redirect_to people_path
+  end
+
+  def create
+    leader = User.find(params[:leader_id])
+    Relationship.create(leader_id: params[:leader_id], follower: current_user) if current_user.can_follow?(leader)
```

Comment:
Why are you passing `leader_id: params[:leader_id]` when you have already leader object?
should be `Relationship.create(leader: leader ...)`